scripts/invalidate_unmodified_studies.py

```
@@ -0,0 +1,105 @@
+        return
+
+    print("\nUnmounting studies...")
+    success_count = 0
+    failure_count = 0
+    for study in tqdm(studies):
+        try:
+            study_uuid = study["elementUuid"]
+            result = invalidate_study(study_uuid)
+            result.raise_for_status()
+            success_count += 1
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)